### DISCUSSION TITLE
Max retries exceeded with url ?

### DISCUSSION THREAD

**Discussion by Anas099X** [devlikeapro/waha#352] [General]
![image](https://github.com/devlikeapro/whatsapp-http-api/assets/99427334/766d2cb1-31c5-4841-81a5-bc8ef88a0aaa)
can anyone help me with that?

**Comment by Anas099X** [comment]
it seems to happen randomly. im using WAHA plus btw

**Comment by allburov** [comment]
H!
Could you give a bit more information?
1. What version do you use (the whole response from GET /api/version would
help)
2. What engine do you use?
3 Do you use files or MongoDB storage?
4. How many sessions do you run?
5. How do you expose the service outside of the world?
6. Is it local connection or over the wire/internet?
7. Is the session you're working with in WORKING status?
8. Do other endpoint like GET /api/version working while swndText doesn't?

It could be some blockers in the code, wanna collect info to make aure it's
the case. We noticed some delays before but it happened only when you start
a session, not after


вт, 28 мая 2024 г., 21:03 Anas099X ***@***.***>:

> it seems to happen randomly. im using WAHA plus btw
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/devlikeapro/whatsapp-http-api/discussions/352#discussioncomment-9581601>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABBMR3XXDMYXB47N7S6HN2DZESFEDAVCNFSM6AAAAABIM35NSCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TKOBRGYYDC>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: <devlikeapro/whatsapp-http-api/repo-discussions/352/comments/9581601@
> github.com>
>


[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by Anas099X** [reply]
No worries, the problem got fixed
